### PR TITLE
Riemann tool to monitor Mesos slaves

### PR DESCRIPTION
**Pull request by ktf**
Contrary to riemann-mesos, this is meant to be run on every single mesos slave.

### CODE REVIEW

File: tools/riemann-mesos-slave/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Riemann Mesos
```
Comment (jamtur01) [inline tools/riemann-mesos-slave/README.md]: Riemann Mesos Slave

**Comment by jamtur01** [comment]
@ktf This seems to have a lot of code overlap with `riemann-mesos` - why not a configuration switch instead of a whole new gem?

**Comment by ktf** [comment]
The hope is that at some point mesos slave metrics endpoints will contain more information (e.g. about tasks) so that the two could possibly diverge.

In principle all we would need is actually a riemann-tool which can query any http and map the results to riemann metrics, then you would basically end up with one single tool and different mapping configurations.

**Comment by jamtur01** [comment]
How about keeping them as a single tool with a switch for now and then split them if they diverge at some point?

**Comment by ktf** [comment]
Ok. Closing this for now. Will reopen using a separate branch as I've more developments for a different plugin I want to push to my repo for tests.

**Pull request closed without merging**